### PR TITLE
added login with google

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import 'babel-polyfill'
 import 'utils/validation/yup'
 
 import React from 'react'
+import { PersistGate } from 'redux-persist/integration/react'
```
Comment (leonprou) [inline dapp/src/index.js:5]: cool, what's this for?

File: server/src/models/StudioUser.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const validator = require('validator')
+const mongoose = require('mongoose')
+const { Schema } = mongoose
+
+const StudioUserSchema = new Schema({
+  picture: { type: String },
+  externalId: { type: String },
+  displayName: { type: String },
+  email: { type: String, required: [true, "can't be blank"], validate: [validator.isEmail, 'invalid email'] },
+  firstName: { type: String, required: 'First Name is required' },
```
Comment (leonprou) [inline server/src/models/StudioUser.js:10]: Name -> name

**Review by leonprou — CHANGES_REQUESTED** [review]
Backend is pretty straightforward,
didn't get to change to dig into the front, but looks clean 👍 
see small comments.

Question is, do we want to merge this to master and then to prod soon?
I have a feeling it's better to deploy this as feature branch and get feature review feedback

File: server/src/models/StudioUser.js (hunk 2 above)
Comment (leonprou) [reply server/src/models/StudioUser.js:10]: and on Second Name too :)

File: dapp/src/index.js (hunk 1 above)
Comment (LiorAgnin) [reply dapp/src/index.js:5]: @leonprou  Help to keep the jwt token persistent.

**Review by leonprou — DISMISSED** [review]

**Pull request closed without merging**